composer/workflows/use_local_deps_test.py

```diff
@@ -12,7 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
 
+import pytest
+
+
+@pytest.mark.skipif(
```

Comment:
Note to self: I can test this on Python 3 by updating the sys.path (PYTHONPATH).

See: composer/workflows/bq_copy_across_locations_test.py added in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1678